.pipelines/java_packaging.yml

```diff
@@ -0,0 +1,37 @@
+parameters:
+- name: JavaPackaging
+  displayName: Package and Publish Java Package
+  type: boolean
+  default: true
+stages:
+- stage: Package and Publish Java Package
+  jobs:
+  - job:
+    workspace:
+      clean: all
+    pool: 'onnxruntime-Win-CPU-2022'
+
+    steps:
+    - script: |
+        ./build.sh -DOCOS_BUILD_JAVA=ON
+      displayName: build the extensions java package
+
+    - task: CopyFiles@2
+      displayName: 'Copy Java Files to Artifact Staging Directory'
```

Comment:
any DLL signing?

Reply:
the package created is just the .jar file so no DLL signing i believe (ORT pipeline also only does ESRP sign for DLLs and NuGets with the respective esrp template yamls, but none for java packages: https://github.com/microsoft/onnxruntime/blob/b84cb247e3ef06639925120a84838ab970ef6843/tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml)